README.md

```diff
@@ -329,6 +329,20 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
 
     Women Hack for Non-Profits (WHFNP) is a volunteer community that enhances the skills of women in technology, supports their development and provides opportunities to explore open source. The group matches skilled women to open source projects that are being developed for non-profits, charities and individuals to create a positive social impact.
 
+  Slack Teams.
```

Comment:
1. Format it as a proper heading, kind of like `### Slack teams'

2. Maybe we should rename the section to something broader, something like chatrooms, or online discussion rooms? We might add more discord servers, gitter rooms, IRC etc.

3. Add a link to this on the table to contents in the top of the README, and add a link to the top of the page right below the heading (copy paste it from another section)

Reply:
For 3, use `([Go to top](#awesome4girls--))`